src/Shouldly.Tests/ShouldContain/LongStringScenario.cs

```diff
@@ -12,7 +12,7 @@ protected override void ShouldThrowAWobbly()
 
         protected override string ChuckedAWobblyErrorMessage
         {
-            get { return "longString should contain \"fff\" but does not"; }
+            get { return "longString should contain \"fff\" but was actually \"aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa\""; }
```

Comment:
We should have a max length on this. `But was "aaaaaaaaaaaaaaaaaaaaaaaaaaa....`. we should use the same length as elsewhere